composer.json

```diff
@@ -1,42 +1,42 @@
 {
-    "name": "opentracing/opentracing",
-    "type": "library",
-    "description": "OpenTracing API for PHP",
-    "license": "Apache-2.0",
-    "authors": [
-        {
-            "name": "José Carlos Chávez",
-            "email": "jcchavezs@gmail.com"
-        }
-    ],
-    "require": {
-        "php": "^5.6 || ^7.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "~5.7.19",
-        "squizlabs/php_codesniffer": "3.*"
-    },
-    "config": {
-        "sort-packages": true
-    },
-    "autoload": {
-        "psr-4": {
-            "OpenTracing\\": "./src/OpenTracing/"
-        },
-        "files": [
-            "./src/OpenTracing/Tags.php",
-            "./src/OpenTracing/Formats.php"
-        ]
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "OpenTracing\\Tests\\": "./tests/OpenTracing"
-        }
+  "name": "opentracing/opentracing",
+  "type": "library",
+  "description": "OpenTracing API for PHP",
+  "license": "Apache-2.0",
+  "minimum-stability": "stable",
+  "authors": [
+    {
+      "name": "José Carlos Chávez",
+      "email": "jcchavezs@gmail.com"
+    }
+  ],
+  "require": {
+    "php": "^7.1"
```

Comment:
Changing the minimum version is the breaking changes. I'm pretty sure users will be surprised. Why did you decide to drop PHP 7.0 support. Could you refer the issue where it was discussed?

Reply:
Here https://gitter.im/opentracing/opentracing-php?at=5c9bd54e36704f6edde2bb99 and following messages. 
Consider also: https://www.php.net/supported-versions.php

Reply:
@piotrooo I'm aware of supported php versions. I just pointed that there is no reference to issue (not chat).

Reply:
@sergeyklay because there isn't any. What do you suggest?

Reply:
Create an issue and describe the problem and the necessary changes. Then add link to the issue in PR description,

Reply:
There was some issue and I linked it.

Reply:
Thank you